containers/kubernetes/reference-content/managed-kubernetes-service-definition.mdx

```diff
@@ -0,0 +1,62 @@
+---
+meta:
+  title: Managed Kubernetes service definition
```

Comment:
```suggestion
  title: Managed Kubernetes service definition
```

Reply:
this seems to be the same?

Reply:
One was in the title and the other in the description.

Reply:
Nevermind. I was reading the previous text, which you had already fixed. I'm sorry.